### PR TITLE
Create My Profile Page and Add Common Components

### PR DESCRIPTION
**Pull request by leeminkyu16**
Hope I didn't mess up lol

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/login/LoginPageView.kt [hunk 1]
```diff
@@ -216,7 +171,7 @@ fun LoginPageView(
 				Button(
 					contentPadding = PaddingValues(1.dp),
 					onClick = {
-						println("Fill")
+				    	println("Fill")
```
Comment (andr-dev) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/login/LoginPageView.kt]: Oops

File: android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt [hunk 2]
```diff
@@ -1,15 +1,296 @@
...
+				Box(
+					modifier = Modifier
+						.padding(30.dp)
+						.clip(CircleShape),
+				) {
+					Box(
+						modifier = Modifier
+							.width(75.dp)
+							.height(75.dp)
+							.background(Color.Gray),
```
Comment (andr-dev) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt:77]: Hmm I'm seeing a lot of 10 multiple spacing and now 75. I think we should stick to using multiples of 8 for >32

**Review by andr-dev — COMMENTED** [review]
Looks good other than non 8 multiple sizings

File: android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt (hunk 2 above)
Comment (leeminkyu16) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt:77]: I fixed it but I feel like you should go complain to the PM and designer instead @AhbedShawarma

Comment (abhaykshar) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt:77]: Lool I'm fine with that as long as it looks good and is consistent throughout

**Review by andr-dev — APPROVED** [review]
LGTM, I'll write a getSpacing function later and we can use that for consistency :)

**Pull request merged**